systemtest/src/main/java/io/strimzi/systemtest/resources/crd/KafkaResource.java

```diff
@@ -242,6 +243,7 @@ public static Kafka createAndWaitForReadiness(Kafka kafka) {
                     if (e.getMessage().contains("object is being deleted")) {
                         return false;
                     } else {
+                        LOGGER.debug("Error found: " + e.getMessage());
```

Comment:
Why you need it here?

Reply:
Because while debugging something, this was missed and not reported at all. I don't remember exact cause anyway. We talked about this with @Frawless - to just add it.

Reply:
So I guess we can add it (in some future PRs) to other resources, right?

Reply:
Yes, I was thinking about the same thing

Reply:
So, should I remove it from here or keep it? I am a bit lost. 
Or the idea is, to add more debug info like this to other resources after parallel tests?

Reply:
Keep it and we'll handle this for other resources after parallel tests